mayastor/src/replica.rs

```diff
@@ -267,6 +273,25 @@ impl Replica {
         Ok(())
     }
 
+    /// Create a snapshot
+    pub fn create_snapshot(
+        self,
+        nvmf_req: *mut spdk_nvmf_request,
+        snapshot_name: &str,
+    ) {
+        let c_snapshot_name = CString::new(snapshot_name).unwrap();
```

Comment:
please use the to_cstring() methods for this as defined in the ffihelper.rs file

Reply:
Fixed.